Basic/ClientDriven/Assets/Scripts/ServerIngredientPhysics.cs

```diff
@@ -14,13 +13,24 @@ public class ServerIngredientPhysics : NetworkBehaviour
 
     public override void OnNetworkObjectParentChanged(NetworkObject parentNetworkObject)
     {
-        SetPhysics(parentNetworkObject == null);
-    }
-
-    void SetPhysics(bool isEnabled)
-    {
-        m_Rigidbody.isKinematic = !isEnabled;
-        m_Rigidbody.interpolation = isEnabled ? RigidbodyInterpolation.Interpolate : RigidbodyInterpolation.None;
-        m_NetworkTransform.InLocalSpace = !isEnabled;
+        // The physic needs to be disabled when the ingredient is parented so that it's following the parent instead of moving freely.
+        if (parentNetworkObject != null)
+        {
+            // Setting isKinematic to true will prevent the object from being affected by the physic in the game,
```

Comment:
As per the company's coding conventions, multi-line comments should be done with `/**/` (applies to other comments as well)